src/app/main.cpp

```diff
@@ -65,6 +72,40 @@ void initGraphics()
 
 int main( int argc, char **argv )
 {
+  qInfo() << "main begins";
+  if ( argc > 1 && strcmp( argv[1], "-service" ) == 0 )
```

Comment:
If I understand correctly, this should be true when someone starts QField with `qfield -service`. Is there a reason for the preference of the BSD style CLI args with single dash in front? If possible, I think double dash would align with pretty much everything other CLI I am dealing with within OPENGIS.ch.